components/Footer.js

```diff
@@ -0,0 +1,41 @@
+import Link from "next/link"
+
+export default function Footer() {
+  return (
+    <div className="grid sm:grid-cols-8 md:grid-cols-10 lg:grid-cols-12 text-white mt-5">
+      <div className="sm:col-span-2 flex items-center justify-center p-2">
+        <img src="/logo.png" className="max-h-24" />
```

Comment:
Do not use \<img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element.  @next/next/no-img-element